### PR TITLE
Issue #9

### LINKED ISSUE #9: API call to add date and hours for a user

### PR DESCRIPTION
**Pull request by parulkhullar**
Added methods to add date and hours for a specific user. Additionally modified tests.py for integrated test methods for the added functionality. closes #9

### CODE REVIEW

**Comment by houngj** [comment]
Good job parul! Here are a few things I would like fixed:
1 - Delete all test created content at the end of your test function
2 - Allow apiUtil.addDateandWorkHours to allow the user variable to be set as a User<object> or a string
3 - Set apiUtil.addDateandWorkHours hours variable default value to 0
suggestion - if hours variable is 0 maybe it would be a good idea to simply not add it to the database. This might save some space if users are trying to habitual in logging their hours, even if they did not workout.

**Comment by parulkhullar** [comment]
I believe that number 3 is done by the database by default

File: simply_fit/simply_fit_api/tests.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 from django.test import TestCase
-from .models import utils, User
+from django.utils import timezone
+from .models import utils, User, workoutLog
 from .views import apiUtil
+import datetime
```
Comment (houngj) [inline simply_fit/simply_fit_api/tests.py]: remove datetime import, it s not being used

File: simply_fit/simply_fit_api/tests.py [hunk 2]
```diff
@@ -14,8 +17,26 @@ def test_create_new_user(self):
...
 		userBSly.delete()
+		self.assertEqual(None, apiUtil.getUser("BSly"))
 	def test_create_existing_user(self):
 		apiUtil.createUser("testCreateExistingUser", "T", "M")
 		self.assertRaises(KeyError, apiUtil.createUser, "testCreateExistingUser", "T", "M")
 		userTmp = apiUtil.getUser("testCreateExistingUser")
-		userTmp.delete()		
+		userTmp.delete()	
+		self.assertEqual(None, apiUtil.getUser("testCreateExistingUser"))
+	def test_addDateAndWorkoutHours(self):
```
Comment (houngj) [inline simply_fit/simply_fit_api/tests.py]: Add comments to intentions of testing

File: simply_fit/simply_fit_api/views.py [hunk 3]
```diff
@@ -11,13 +13,45 @@ def createUser(userName, firstName, lastName):
...
 		try:
 			user = User.objects.get(userName=user)
 			return user
 		except:
 			return None
+
+	''' This method add the date and workout hours for a specified user
+		addDateAndWorkoutHours:
+			@params : username: [username; firstName; lastName]
+					  date: dateTime
```
Comment (houngj) [inline simply_fit/simply_fit_api/views.py]: You are using a django.util.timezone as variable type in database. Please match type.

File: simply_fit/simply_fit_api/views.py [hunk 4]
```diff
@@ -1,8 +1,10 @@
 from django.shortcuts import render
 from django.http import HttpResponse
-from .models import User
+from .models import User, workoutLog
+import datetime
```
Comment (houngj) [inline simply_fit/simply_fit_api/views.py]: datetime is not being called here either, please remove

File: simply_fit/simply_fit_api/views.py [hunk 5]
```diff
@@ -11,13 +13,45 @@ def createUser(userName, firstName, lastName):
...
 			return None
+
+	''' This method add the date and workout hours for a specified user
+		addDateAndWorkoutHours:
+			@params : username: [username; firstName; lastName]
+					  date: dateTime
+					  Hours: integer
+	'''
+	@staticmethod
+	def addDateAndWorkoutHours(user, date, hours):
```
Comment (houngj) [inline simply_fit/simply_fit_api/views.py]: add comments to all methods your wrote

**Review by houngj — CHANGES_REQUESTED** [review]
Please review my comments

**Review by houngj — APPROVED** [review]

**Pull request merged**